core/src/main/java/io/grpc/ServerBuilder.java

```diff
@@ -41,6 +41,11 @@
  * @param <T> The concrete type of this builder.
  */
 public abstract class ServerBuilder<T extends ServerBuilder<T>> {
+
+  public static ServerBuilder<? extends ServerBuilder<?>> forAddress(String name, int port) {
```

Comment:
Just like for client-side, I think it is equally expressive to just have `ServerBuilder<?>` here.